### PR TITLE
comment all log format

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <764524258@qq.com>

### CODE REVIEW

**Comment by daixiang0** [comment]
@FUSAKLA mind have a look?

File: cmd/thanos/main.go [hunk 1]
```diff
@@ -46,7 +46,7 @@ func main() {
 
 	logLevel := app.Flag("log.level", "Log filtering level.").
 		Default("info").Enum("error", "warn", "info", "debug")
-	logFormat := app.Flag("log.format", "Log format to use.").
+	logFormat := app.Flag("log.format", "Log format to use, support logfmt or json").
```
Comment (bwplotka) [inline cmd/thanos/main.go]:
```suggestion
	logFormat := app.Flag("log.format", "Log format to use. Possible options: logfmt or json").
```

Comment (FUSAKLA) [reply cmd/thanos/main.go]: We might do this event for the log level

**Comment by daixiang0** [comment]
ping @FUSAKLA

**Pull request merged**